src/OperationalInsights/OperationalInsights/Workspaces/NewAzureOperationalInsightsWorkspaceCommand.cs

```diff
@@ -71,6 +71,9 @@ public class NewAzureOperationalInsightsWorkspaceCommand : OperationalInsightsBa
             HelpMessage = "Allow to opt-out of local authentication and ensure customers can use only MSI and AAD for exclusive authentication")]
         public bool? DisableLocalAuth;
 
+        [Parameter(Mandatory = false, HelpMessage = "The resource ID of the default Data Collection Rule to use for this workspace. Expected format is - /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/dataCollectionRules/{dcrName}.")]
+        public string DefaultDataCollectionRuleResourceId;
```

Comment:
I don't see why it's called default

Reply:
please see the official documentation: 
https://review.docs.microsoft.com/en-us/rest/api/loganalytics/workspaces/create-or-update?branch=main

![image](https://user-images.githubusercontent.com/74346518/173220161-98bfe68c-5e02-4651-b741-f16e8b01abf0.png)